services/app-api/forms/sar.json

```diff
@@ -107,9 +853,436 @@
+                }
+              },
+              {
+                "id": "ret_deathNumberOfAdults",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of Older Adults",
+                  "heading": "Death"
+                }
+              },
+              {
+                "id": "ret_deathNumberOfPD",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of PD"
+                }
+              },
+              {
+                "id": "ret_deathNumberOfIDD",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of I/DD"
+                }
+              },
+              {
+                "id": "ret_deathNumberOfMHSUD",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of MH/SUD"
+                }
+              },
+              {
+                "id": "ret_voluntaryDisenrollmentNumberOfAdults",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of Older Adults",
+                  "heading": "Voluntary disenrollment"
+                }
+              },
+              {
+                "id": "ret_voluntaryDisenrollmentNumberOfPD",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of PD"
+                }
+              },
+              {
+                "id": "ret_voluntaryDisenrollmentNumberOfIDD",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of I/DD"
+                }
+              },
+              {
+                "id": "ret_voluntaryDisenrollmentNumberOfMHSUD",
+                "type": "number",
+                "validation": "number",
+                "props": {
+                  "label": "Number of MH/SUD"
+                }
+              },
+              {
+                "id": "ret_otherReasons",
+                "type": "checkbox",
+                "validation": "checkbox",
+                "props": {
+                  "label": "Other reasons",
+                  "hint": "Check all that apply.",
+                  "heading": "Other",
+                  "choices": [
+                    {
+                      "id": "2VffASWS2XRfAlc3uLzxCVAC",
+                      "label": "Moved out of MFP jurisdiction/state/territory",
+                      "children": [
+                        {
+                          "id": "otherReasons_movedOutNumberOfOlderAdults",
+                          "type": "number",
+                          "validation": {
+                            "type": "number",
```

Comment:
these nested fields are type: number but not erroring when I enter something invalid. likely not an issue with validation, but perhaps with when/how validation is triggering? might be out of scope for this PR

Reply:
I don't think we've got validation set up just yet for this form! that might be the issue?